validator/src/test/java/com/amazon/aoc/validators/LoadBalancingValidatorTest.java

```diff
@@ -47,26 +48,35 @@ public void testValidationSucceeds() throws Exception {
     validationConfig.setCallingType("http");
     Context context = initContext();
 
-    String mockedTrace =
-        "[{ \"segments\": [{ \"document\": '{ \"metadata\": { \"default\": {\"collector-id\": 3} }, \"subsegments\": [{ \"metadata\": { \"default\": {\"collector-id\": 3} }}, { \"metadata\": { \"default\": {\"collector-id\": 3} }}] }' }] }]";
+    ObjectMapper mapper = new ObjectMapper();
+    JsonNode jsonNode =
+        mapper.readValue(
+            new File(
+                "./src/test/java/com/amazon/aoc/testdata/LoadBalancingValidatorTraceSuccess.json"),
+            JsonNode.class);
+    String mockedTrace = mapper.writeValueAsString(jsonNode);
 
-    ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
+    // String mockedTrace = "[{ \"segments\": [{ \"document\": '{ \"metadata\": { \"default\":
+    // {\"collector-id\": 3} }, \"subsegments\": [{ \"metadata\": { \"default\": {\"collector-id\":
+    // 3} }}, { \"metadata\": { \"default\": {\"collector-id\": 3} }}] }' }] }]";
+
+    mapper = new ObjectMapper(new YAMLFactory());
     List<Trace> mockedTraceList =
         mapper.readValue(
             mockedTrace.getBytes(StandardCharsets.UTF_8), new TypeReference<List<Trace>>() {});
 
     runValidation(validationConfig, context, mockedTraceList);
   }
 
-  /** test validation when fails due to difference collector ids */
+  /** test validation when it fails due to different collector ids */
   @Test
   public void testValidationFailedExpectedMatchingId() throws Exception {
     ValidationConfig validationConfig = new ValidationConfig();
     validationConfig.setCallingType("http");
     Context context = initContext();
 
     String mockedTrace =
-        "[{ \"segments\": [{ \"document\": '{ \"metadata\": { \"default\": {\"collector-id\": 3} }, \"subsegments\": [{ \"metadata\": { \"default\": {\"collector-id\": 2} }}, { \"metadata\": { \"default\": {\"collector-id\": 3} }}] }' }] }]";
+        "[{ \"segments\": [{ \"document\": '{ \"metadata\": { \"default\": {\"collector-id\": 2} }, \"subsegments\": [{ \"metadata\": { \"default\": {\"collector-id\": 3} }}, { \"metadata\": { \"default\": {\"collector-id\": 3} }}] }' }] }]";
```

Comment:
Shouldn't this be stored in it's own file also?

Reply:
Changed in my newest commit.